mkdocs/index.md

```diff
@@ -213,25 +233,62 @@ template: home.html
       })
 
       watch(checkboxesApps, (newVal, oldVal) => {
+        let item_tags_lowercased = []
```

Comment:
There is quite complicated logic implemented. Please describe it in comments at the beginning of "watch" function.